conduct/COC_POLICY.md

```diff
@@ -0,0 +1,99 @@
+This document defines the OpenJS Foundation's code of conduct policy.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Policy
+
+The OpenJS Foundation (OpenJSF) delegates to the Cross Project Council (CPC), via its [charter][], the responsibility of choosing, maintaining, and enforcing a Code of Conduct (CoC) for all of its projects, collab spaces, and for any related activities.
+
+## Code of Conduct
+
+The OpenJSF's code of conduct is located at [code-of-conduct.openjsf.org][CoC]. It is based on the [Contributor Covenant](https://www.contributor-covenant.org/).
+
+A common code of conduct allows a common escalation and appeal process and allows better support of projects in their efforts to apply the code of conduct.
+
+All OpenJSF projects, collab spaces, and any activities related to the OpenJSF are operating under this code of conduct unless otherwise stated. (For example, events organized in partnership with other organizations might be operating under a different code of conduct.)
+
+Projects joining the foundation that already have a code of conduct will need to adopt the OpenJSF's CoC within three (3) months. Projects that don't already have a code of conduct operate under the OpenJSF's CoC as soon as they enter incubation.
+
+## Reporting incidents
+
+Reporting incidents should be as simple and flexible as possible.
+
+All information necessary to report an incident must be publicly documented directly alongside the code of conduct itself. This information must contain at least the following:
+
+1. A general email address at which incidents can be reported.
+2. The name and email address of each recipient of the general email address.
+
+Reporters may choose to report an incident by using the general email address provided or by contacting any of its recipients individually, either by using their provided email, reaching out in person, or by any other reasonable means.
+
+An acknowledgment of receipt should be promptly issued to incident reporters.
+
+The name and contact details of the people responsible for managing the incident report should be provided to reporters as soon as possible (to uncover conflicts of interest upfront and allow for escalation).
+
+## Commitment
+
+The OpenJSF makes the following commitments to reporters and victims of CoC incidents:
+
+1. All incident reports will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.
+2. The identity of the victim(s) and reporter(s) of a code of conduct violation will be kept confidential.
```

Comment:
Confidential from whom? Below it's mentioned that the CoC Team "is empowered to involve external people". Will the identities be kept confidential from them as well?

Reply:
I'm splitting those commitments up to make them more explicit. To answer your comment, however, external people are required to abide by the policy as stated, so that's covered.

Reply:
Two separate thoughts here:
1. This means that someone reporting an incident cannot know ahead of time the set of people that their identity will be revealed to, on account of the CoC Team being empowered to bring in pretty much anyone. I guess that's ok?
2. There is no mention here of keeping the identity of someone accused of a violation confidential. Should that not be the case at least until the response has been determined?

Reply:
> This means that someone reporting an incident cannot know ahead of time the set of people that their identity will be revealed to, on account of the CoC Team being empowered to bring in pretty much anyone. I guess that's ok?

Yeah. Imagine there's an issue where the CoC Team needs to bring in a translator. There's no reasonable way to know their identity beforehand. It also doesn't matter. We're trying to avoid the case where someone reports a problem caused by someone and unknowingly report it to the very person that's causing the issue. And the current policy covers that imho.

> There is no mention here of keeping the identity of someone accused of a violation confidential. Should that not be the case at least until the response has been determined?

Yeah, that's a little more complicated. There are circumstances where that can't be the case (for example, if there's a risk of harm to additional people, legal considerations, etc.). Not sure how to properly manage this. Suggestions welcomed.

Reply:
I've added your second point as a task that needs to be resolved before merging this PR. Resolving this.